frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/pipeline-metrics-utils.ts

```diff
@@ -51,7 +54,7 @@ const formatPositiveValue = (v: number): string =>
 export const formatValue = (v: number): string =>
   (v < 0 ? '-' : '') + formatPositiveValue(Math.abs(v));
 export const formatDate = (date: Date) => {
-  return `${moment(date).format('MMM DD')}`;
+  return dateFormatter.format(date);
```

Comment:
@spadgett We lost our x-axis:

![image](https://user-images.githubusercontent.com/8126518/113015050-25578700-914b-11eb-8f5d-19dad046969b.png)

My quick debugging is this change.

Reply:
Let me know if you need anything else here @spadgett -- I'll step away from this PR until you need a re-review or help testing it.

Reply:
Thanks @andrewballantyne, I have a fix.

<img width="702" alt="Screen Shot 2021-04-15 at 10 30 18 AM" src="https://user-images.githubusercontent.com/1167259/114891675-1410aa80-9dda-11eb-8f09-b2c3adf18c71.png">